app/spec/services/github_checks_verifier_spec.rb

```diff
@@ -87,16 +91,27 @@
     end
   end
 
-  describe "#fail_unless_all_success" do
-    it "raises an exception if some check is not successful" do
+  describe "#fail_unless_all_conclusions_allowed" do
+    it "raises an exception if some check conclusion is not allowed" do
       all_checks = [
         OpenStruct.new(name: "test", status: "completed", conclusion: "success"),
         OpenStruct.new(name: "test", status: "completed", conclusion: "failure")
       ]
 
       expect do
-        service.fail_unless_all_success(all_checks)
-      end.to raise_error(StandardError, "One or more checks were not successful, exiting...")
+        service.fail_unless_all_conclusions_allowed(all_checks)
+      end.to raise_error(StandardError, 'The conclusion of one or more checks were not allowed. Allowed conclusions are: success, skipped. This can be configured with the \'allowed-conclusions\' param.')
+    end
+
+    it "does not raise an exception if all checks conlusions are allowed" do
+      all_checks = [
+        OpenStruct.new(name: "test", status: "completed", conclusion: "success"),
+        OpenStruct.new(name: "test", status: "completed", conclusion: "skipped")
+      ]
+  
+      expect do
+        service.fail_unless_all_conclusions_allowed(all_checks)
+      end.not_to raise_error()
```

Comment:
You can remove the empty parenthesis 
```suggestion
      end.not_to raise_error
```